packages/web-console/src/providers/SettingsProvider/types.ts

```diff
@@ -0,0 +1,30 @@
+export type Settings = Partial<{
+  "acl.oidc.enabled": boolean
+  "acl.oidc.client.id": string
+  "acl.oidc.host": string
+  "acl.oidc.port": number
+  "acl.oidc.tls.enabled": boolean
+  "acl.oidc.authorization.endpoint": string
+  "acl.oidc.token.endpoint": string
+  "acl.oidc.pkce.required": boolean
+  "acl.basic.auth.realm.enabled"?: boolean
+  "questdb.type"?: "OSS" | "EE"
+  "questdb.version"?: string
```

Comment:
why are questdb.type and questdb.version optional?
i would understand if the acl fields are made optional, they are only present in EE.